scripts/qa_crowbarsetup.sh

```diff
@@ -395,6 +395,16 @@ function addcloud6pool
     add_mount "SUSE-OpenStack-Cloud-6-Pool" $clouddata':/srv/nfs/repos/SUSE-OpenStack-Cloud-6-Pool/' "$tftpboot_repos12sp1_dir/SUSE-OpenStack-Cloud-6-Pool/" "cloudpool"
 }
 
+function addcloud7pool
+{
+    add_mount "SUSE-OpenStack-Cloud-7-Pool" $clouddata':/srv/nfs/repos/SUSE-OpenStack-Cloud-7-Pool/' "$tftpboot_repos12sp2_dir/SUSE-OpenStack-Cloud-7-Pool/" "cloudpool"
+}
+
+function addcloud7maintupdates
+{
+    add_mount "SUSE-OpenStack-Cloud-7-Updates" $clouddata':/srv/nfs/repos/SUSE-OpenStack-Cloud-7-Updates/' "$tftpboot_repos12sp2_dir/SUSE-OpenStack-Cloud-7-Updates/" "cloudmaintup"
```

Comment:
this is wrong, there is an $arch missing, I think it should be /srv/nfs/repos/x86_64/...